packages/xpub-lib/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@swan-bitcoin/xpub-lib",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
this is technically a breaking change for users deriving taproot addresses? Should we bump the minor version instead?

Reply:
yes :sob: